terraform/bigquery.tf

```diff
@@ -462,14 +462,14 @@ resource "google_bigquery_routine" "unique_events_by_date_type" {
       received,
       event,
       payload,
-      JSON_VALUE(payload, "$.organization.login") organization,
-      SAFE_CAST(JSON_VALUE(payload, "$.organization.id") AS INT64) organization_id,
-      JSON_VALUE(payload, "$.repository.full_name") repository_full_name,
-      SAFE_CAST(JSON_QUERY(payload, "$.repository.id") AS INT64) repository_id,
-      JSON_VALUE(payload, "$.repository.name") repository,
-      JSON_VALUE(payload, "$.repository.visibility") repository_visibility,
-      JSON_VALUE(payload, "$.sender.login") sender,
-      SAFE_CAST(JSON_QUERY(payload, "$.sender.id") AS INT64) sender_id,
+      LAX_STRING(payload.organization.login) organization,
```

Comment:
Why not SAFE.STRING(...)?

Reply:
To keep the behavior the same to how the existing query treated non-scalar values. This will give null while SAFE.STRING would give an error.